lib/haj_web/live/applications_live.ex

```diff
@@ -89,13 +91,34 @@ defmodule HajWeb.ApplicationsLive do
                 </tr>
                 <tr x-show="show" class={if rem(i, 2) == 0, do: "bg-white", else: "bg-gray-50"}>
                   <td colspan="5" class="w-full gap-2 px-4 py-3">
-                    <div>Tid för ansökan: <%= application.inserted_at %></div>
-                    <div>Övrigt: <%= application.other %></div>
-                    <div>Eventuell rangordning: <%= application.ranking %></div>
+                    <div><b>Tid för ansökan:</b> <%= application.inserted_at %></div>
+                    <hr class="mb-2"/>
+                    <div>
+                      <p><b>Övrigt</b></p>
+                      <%= case application.other do
```

Comment:
This is some old code, but I think it should be possible to do with css, `class="whitespace-pre-line"` should do the trick. Also, it is possible to do something similar to:
```
<div :if={application.other == nil}>Inget svar</div>
```
Instead of the case statement :)

Reply:
Nice! Thanks! Don't know much elixir, so that is a nice tip. 

Without more insight in what is idiomatic, I would say that I like `case` more, since it makes it clear that all cases are handled and that content for multiple cases won't appear at the same time (which I feel can be an issue sometimes in react at least when using conditionals similar to this way (no idea how well this applies for phoenix though))

`whitespace-pre-line` seems to work :stuck_out_tongue:, that is probably a better solution then

Reply:
This is inside of `heex`, and I think it is ideomatic to keep it looking as close to html and to use `:if`, `:for`, etc. Feels wierd to escape out of `heex` with `<%= =>` only to have `heex` inside of it via `~H`

Reply:
Reasonable! I agree